spec/factories.rb

```diff
@@ -1,5 +1,6 @@
 FactoryBot.define do
   factory :content_change do
+    id { SecureRandom.uuid }
```

Comment:
Is this needed for build? Since it's set automatically on create I'm just wondering if this was added to resolve an issue.

Reply:
Removed